test/unit/setting_manager_test.rb

```diff
@@ -83,4 +83,48 @@ class SettingManagerTest < ActiveSupport::TestCase
       end
     end
   end
+
+  describe 'Validations' do
+    setup do
+      @validation_backup = Setting._validators[:value].dup
+    end
+
+    teardown do
+      Setting._validators[:value] = @validation_backup
+    end
```

Comment:
Settings in our tests is in terrible shape - we have fixtures plus stubs. Would be great to have some definitive solution, for example if we would extend minitest/rspec so every test settings get automatically reloaded. Out of scope for this PR tho.

Reply:
All of the DSL defined settings gets loaded and it's default value will be automatically used. I believe that finishing this, migrate all settings to use the DSL and dropping the fixtures all together is the best path forward here :)